### PR TITLE
feat: add frontend-low-code docs

### PR DESCRIPTION
**Pull request by Jandiasnow**
ref: https://github.com/kubebb/website/issues/38

### CODE REVIEW

File: docs/lowcode-development/development/frontend-low-code.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
+---
+sidebar_position: 2
+---
+
+# 前端层基于低码 Kit 的开发
+
+本章节介绍如何使用低码 Kit 开发前端应用。
+
+
```
Comment (bjwswang) [inline docs/lowcode-development/development/frontend-low-code.md]: @Jandiasnow twoo much blanks lines here.

File: docs/lowcode-development/development/frontend-low-code.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+## 1. 新增应用
+首先我们需要在云梯工作台【应用管理】-【新增应用】下创建一个应用，如果您已有可用的应用则可以跳过本步，如下图所示，我们创建一个名为 `Hello World` 的空白应用：
```
Comment (bjwswang) [inline docs/lowcode-development/development/frontend-low-code.md:11]: add a blank line here .

**Review by bjwswang — CHANGES_REQUESTED** [review]
@Jandiasnow Please fix the comments post by markdownlint bot

File: docs/lowcode-development/development/frontend-low-code.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+接下来我们需要在云梯工作台【分支管理】-【创建分支】创建一个开发分支，如果您已有相关分支可以跳过本步，如下图所示，我们创建一个名为 `dev` 的分支：
+:::info
+可在【页面管理】- 点击右上角切换到 `dev` 分支
+:::
+![创建分支](images/branches_create.png)
+## 4. 新增页面
+
+点击【页面管理】，点击【右上角选择分支】切换到 `dev` 分支，如图所示，
+
+接下来我们需要创建一个页面来实现 `Hello World` 的功能，如下所示，当前我们还没有页面模板（已有的页面），可以不选择：
```
Comment (Carrotzpc) [inline docs/lowcode-development/development/frontend-low-code.md:57]: 与图中命名最好一致 `HelloWorld`，图中的命名多了一个 `w`。。。

**Comment by Carrotzpc** [comment]
@bjwswang 看下，应该没问题了

**Review by bjwswang — APPROVED** [review]

File: docs/lowcode-development/development/frontend-low-code.md [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+
+   ![添加全局函数](images/global_add_func.png)
+
+  3. 绑定 sdk 中的请求方法：
+
+     - 方式一：可以通过 `this.utils.bff.getXXX` 在 【源码面板】中来调用 `sdk` 中的请求方法
+
+     - 方式二：【属性设置面板】-【调用函数】（目前仅支持解析全局函数 `bff` 所绑定的 npm 资产的方法 ），如下图所示：
+
+    ![添加页面数据](images/page_add_api.png)
```
Comment (bjwswang) [inline docs/lowcode-development/development/frontend-low-code.md]: @Jandiasnow

File: docs/lowcode-development/development/frontend-low-code.md (hunk 3 above)
Comment (bjwswang) [reply docs/lowcode-development/development/frontend-low-code.md:57]: @Jandiasnow

**Pull request merged**